site/docs/spark-structured-streaming.md

```diff
@@ -24,7 +24,21 @@ As of Spark 3.0, DataFrame reads and writes are supported.
 
 | Feature support                                  | Spark 3.0| Spark 2.4  | Notes                                          |
 |--------------------------------------------------|----------|------------|------------------------------------------------|
-| [DataFrame write](#writing-with-streaming-query) | ✔        | ✔          |                                                |
+| [DataFrame write](#writing-with-streaming-query) | ✔        | ✔         |                                                |
```

Comment:
What changed on this line? Can we roll that back?

Reply:
Just for alignment. Rolled back.